packages/renderer/src/lib/preferences/PreferencesConnectionCreationOrEditRendering.svelte

```diff
@@ -93,12 +93,14 @@ let contextsUnsubscribe: Unsubscriber;
 const buttonLabel = connectionInfo ? 'Update' : 'Create';
 const operationLabel = connectionInfo ? 'Update' : 'Creation';
 let loggerHandlerKey: symbol | undefined = undefined;
+let reconnectedUI: boolean = false;
 
 // reconnect the logger handler
 $effect(() => {
-  if (logsTerminal && loggerHandlerKey) {
+  if (!reconnectedUI && logsTerminal && loggerHandlerKey) {
     try {
       reconnectUI(loggerHandlerKey, getLoggerHandler());
+      reconnectedUI = true;
```

Comment:
suggestion: Could we have the reconnectedUI in the finally ? the problem may occurs, if the reconnectedUI throw an error, we would be in a weird state, having it in the finally would probably avoid that